spec/models/user_spec.rb

```diff
@@ -2,4 +2,16 @@
 
 RSpec.describe User, type: :model do
   pending "add some examples to (or delete) #{__FILE__}"
+
+  describe "#admin?" do
+    it "is not an admin if the role is not admin" do
+      user = FactoryBot.create(:user, role: 'member')
+      expect(user.admin?).to eq(false)
+    end
+
+    it "is an admin if the role is admin" do
+      user = FactoryBot.create(:user, role: 'admin')
+      expect(user.admin?).to eq(true)
+    end
+  end
 end
```

Comment:
Firstly, this whole test is pending, and thus is not valid.
Secondly, we may need to remove this since there is no admin features as of yet

Reply:
Removed admin test